src/routes/pages/de/guides/sweep-to-self-custody.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Sende Sats zu deiner selbstverwalteten Wallet
+description: In diesem Guide werden wir eine einfache Lösung vorstellen, wie du das Gegenparteirisiko reduzieren und deine Sats auf ein Self-Custody-Wallet ziehen kannst.
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
@Flobstr a few of the guides have this super long set of dashes. Can you change these lines to be just 3 dashes.